src/css/cssCastUnit.js

```diff
@@ -0,0 +1,60 @@
+define([
+	"../core",
+	"../var/rfxnum"
+], function( jQuery, rfxnum ) {
+
+function cssCastUnit( elem, prop, parts, tween ) {
```

Comment:
Variables here could have better names, especially now that this code is a fully-fledged function serving two purposes. I recommend something like:
* cssCastUnit → adjustCSS
* parts → value
* getCSS → currentValue
* target → initial
* start → recasted
* end → adjusted